packages/riipen-ui/src/components/ProgressBar.jsx

```diff
@@ -78,6 +80,22 @@ class ProgressBar extends React.Component {
             background: ${theme.palette.primary.main};
           }
 
+          .light {
+            background: ${theme.palette.grey[50]};
```

Comment:
same colour as the default fafafa, just changed to be consistent with the other colour